coordinator/internal/authority/userapi_test.go

```diff
@@ -253,32 +253,35 @@ func TestRecovery(t *testing.T) {
 	var seed [32]byte
 	var salt [32]byte
 	testCases := []struct {
-		name     string
-		seed     []byte
-		salt     []byte
-		wantCode codes.Code
+		name        string
+		seed        *[]byte
+		salt        *[]byte
```

Comment:
Since `nil` is a valid slice, why not use `[]byte` as the type. The default initialization is also nil:
```go
package main

import "fmt"

func main() {
	var s []int

	fmt.Println(s==nil) // prints true
}
```

Reply:
The idea is to decide whether to override the value in the test body based on this pointer being set or nil, and I want to be able to override to nil.